src/libraries/System.Net.Quic/ref/System.Net.Quic.cs

```diff
@@ -15,6 +15,7 @@ public partial class QuicClientConnectionOptions : System.Net.Quic.QuicOptions
     }
     public sealed partial class QuicConnection : System.IDisposable
     {
+        public static bool IsSupported { get { throw null; } }
```

Comment:
We don't want to put this on connection.

Reply:
Shouldn't this go through API review first?

Reply:
We'll be doing API reviews for S.N.Quic in big batches once we settle on the shape. It's a work in progress ATM.
But as I said, we're not adding `IsSupported` to `QuicConnection`, that's not the goal of the original issue.

Reply:
> that's not the goal of the original issue.

It isn't? I am probably missing some context because the original issue contained exactly this API proposal...

```cs
namespace System.Net.Quic
{
    public class QuicConnection
    {
        public static bool IsSupported { get; }
    }
}
```

Reply:
I see, I will fix `QuicImplementationProvider.IsSupported` then